partiql-plan/src/main/resources/partiql_plan.ion

```diff
@@ -78,6 +78,25 @@ arg::[
   }
 ]
 
+// Exclude expr and step
+exclude_expr::{
+  root: string,
+  root_case: case,
+  steps: list::[exclude_step],
+}
```

Comment:
Same question as `partiql_ast`. Why not just path?

Reply:
(same as above comment in `partiql_ast`)